Dockerfile

```
@@ -80,10 +80,18 @@ ARG TARGETARCH
 ARG BPFTOOL_TAG=v7.2.0-snapshot.0
 RUN curl -L https://github.com/libbpf/bpftool/releases/download/${BPFTOOL_TAG}/bpftool-${BPFTOOL_TAG}-${TARGETARCH}.tar.gz | tar xz && chmod +x bpftool
 
+# Get bash-completion manifests and generate tetra CLI bash
+# autocompletions (we don't want all bash-completions in the base-build)
+FROM docker.io/library/alpine:3.20.3@sha256:beefdbd8a1da6d2915566fde36db9db0b524eb737fc57cd1367effd16dc0d06d AS cli-autocomplete
+COPY --from=tetragon-builder /go/src/github.com/cilium/tetragon/tetra /usr/bin/
+RUN apk add --no-cache bash-completion && \
+    tetra completion bash | tee /etc/bash_completion.d/tetra > /dev/null && \
```

nit: maybe you don't need tee if you redirect stdout to dev null?
```suggestion
    tetra completion bash > /etc/bash_completion.d/tetra && \
```

---

Right, good point! Thanks. I committed & squashed the suggestion.